src/css/SingleBlog.css

```
@@ -0,0 +1,443 @@
+  font-weight: bold;
+}
+
+.comment-date {
+  font-size: 14px;
+  color: #555;
+  margin-top: 4px;
+  margin-bottom: 10px;
+}
+
+.comment-text {
+  font-size: 18x;
```

Biome (1.9.4)

[error] Unexpected unknown unit: x

See MDN web docs for more details.
Use a known unit instead, such as:


(lint/correctness/noUnknownUnit)